tensorboard/webapp/routes/dashboard_deeplink_provider.ts

```diff
@@ -124,7 +124,7 @@ export class DashboardDeepLinkProvider extends DeepLinkProvider {
       ),
       store.select(selectors.getRunUserSetGroupBy).pipe(
         map((groupBy) => {
-          if (!groupBy) return [];
+          if (!groupBy || groupBy.key === null) return [];
```

Comment:
This now can be reverted as `groupBy.key` cannot be a null.

Reply:
Ah, thanks for catching this. Done